transformer_engine/common/gemm/cublaslt_gemm.cu

```diff
@@ -135,6 +138,17 @@ void cublas_gemm(const Tensor *inputA,
                                                      CUBLASLT_MATMUL_DESC_B_SCALE_POINTER,
                                                      &B_scale_inverse,
                                                      sizeof(B_scale_inverse)));
+    if (D_type == CUDA_R_8F_E4M3 || D_type == CUDA_R_8F_E5M2) {
+        NVTE_CHECK_CUBLAS(cublasLtMatmulDescSetAttribute(operationDesc,
+                                                         CUBLASLT_MATMUL_DESC_D_SCALE_POINTER,
+                                                         &D_scale,
+                                                         sizeof(D_scale)));
+        NVTE_CHECK_CUBLAS(cublasLtMatmulDescSetAttribute(operationDesc,
+                                                         CUBLASLT_MATMUL_DESC_AMAX_D_POINTER,
+                                                         &D_amax,
+                                                         sizeof(D_amax)));
+        NVTE_CHECK_CUBLAS(cublasLtMatrixLayoutCreate(&Cdesc, bias_type, m, n, ldd));
```

Comment:
You are leaking memory here since Cdesc was already created.

Reply:
Thanks for pointing it out. Will fix this.